project.mml

```diff
@@ -287,12 +292,16 @@ Layer:
           WHEN highway IN ('motorway_link', 'trunk_link', 'primary_link', 'secondary_link', 'tertiary', 'tertiary_link', 'residential', 'unclassified', 'road', 'living_street') THEN 'minorroad'
           WHEN highway IN ('service', 'track') THEN 'service'
           WHEN highway='cycleway' THEN 'cycle'
-          WHEN highway IN ('path', 'footway', 'pedestrian', 'steps', 'bridleway') THEN CASE WHEN bicycle IN ('yes', 'designated') THEN 'cycle' ELSE 'pedestrian' END
+          WHEN highway IN ('path', 'footway', 'pedestrian', 'bridleway') THEN CASE WHEN bicycle IN ('yes', 'designated') THEN 'cycle' ELSE 'pedestrian' END
```

Comment:
Shouldn't this test with bicycle also be done in bridge and tunnels?

Reply:
We shouldn't mixed pedestrian, bridleway and footway with cycleway rendering, they could become blue if they are bicycle friendly though.